templates/textfield.php

```diff
@@ -1,8 +1,16 @@
+<?php
+/**
+ * Template file for a textfield
+ *
+ * @package Templates
+ */
+
+?>
 <input
 	class="fm-element"
-	type="<?php echo esc_attr( $this->input_type ) ?>"
+	type="<?php echo esc_attr( $this->input_type ); ?>"
 	name="<?php echo esc_attr( $this->get_form_name() ); ?>"
 	id="<?php echo esc_attr( $this->get_element_id() ); ?>"
 	value="<?php echo esc_attr( $value ); ?>"
-	<?php echo $this->get_element_attributes(); ?>
-/>
\ No newline at end of file
+	<?php echo $this->get_element_attributes(); // Escaped in function call WPCS XSS okay. ?>
```

Comment:
Same again: [There is a style for this already in `master`](https://github.com/alleyinteractive/wordpress-fieldmanager/blob/master/templates/datepicker.php#L18); don't know which is better, but could be consistent.